content/terraform/v1.12.x/docs/language/import/index.mdx

```diff
@@ -1,166 +1,145 @@
-  name = "hashi"
-  # (other resource arguments...)
-}
-```
+1. Define a destination resource configuration for each resource you want to import. 
+1. Add a corresponding `import` block for each existing resource you want to import. 
+1. Run `terraform plan` and verify that the import plan meets your needs. 
+1. Apply the configuration to import the resources and update your Terraform state.
 
-The above `import` block defines an import of the AWS instance with the ID "i-abcd1234" into the `aws_instance.example` resource in the root module.
+You can remove `import` blocks from your configuration after importing resources, but we recommend keeping them as an historical artifact. Refer to [Post import tasks](#post-import-tasks) for more information.
 
-The `import` block has the following arguments:
- - `to` - The instance address this resource will have in your state file.
- - `id` - A string with the [import ID](#import-id) of the resource, or an expression that evaluates to a string. This argument is mutually exclusive with `identity`.
- - `identity` - A resource identity object that uniquely identifies a resource. It is mutually exclusive with `id`.
- - `provider` (optional) - An optional custom resource provider, see [The Resource provider Meta-Argument](/terraform/language/meta-arguments/resource-provider) for details.
+## Define a destination resource 
 
-If you do not set the `provider` argument, Terraform attempts to import from the default provider. 
+Terraform reconciles your configuration and state to create an execution plan. As a result, you must define a `resource` block for any resource in state to prevent Terraform from destroying it.  The only required arguments to the `resource` block for an imported resource are the inline resource type and resource label, which form the Terraform state address. 
 
-### Import ID
+The following example resource is the destination for an AWS instance with a state address of `aws_instance.example`:
 
-The import block's `id` argument can be a literal string of your resource's import ID, or an expression that evaluates to a string, such as `var.instance_id`. Terraform needs this import ID to locate the resource you want to import.
+```hcl
+resource "aws_instance" "example" {
+  name = "renderer"
+}
+```
 
-The import ID must be known at plan time for planning to succeed. If the value of `id` is only known after apply, `terraform plan` will fail with an error.
+You should include provider-specific resource arguments that have non-default values to prevent Terraform from destroying the imported resource on the next apply operation. Terraform uses default values for arguments you do not include in the `resource` block. If Terraform assigns default values, but the existing resource has non-default attributes, the resource in state will not match the actual infrastructure. Refer to the provider documentation for information about which arguments the resource you are importing supports.
```

Comment:
```suggestion
You should include provider-specific resource arguments that have non-default values to prevent Terraform from destroying the imported resource on the next apply operation. Terraform uses default values for arguments you do not include in the `resource` block. If Terraform assigns default values, but the existing resource has non-default attributes, the resource in state will not match the actual infrastructure, and Terraform will plan to update the resource on the next apply operation. Refer to the provider documentation for information about which arguments the resource you are importing supports.
```

Something like this, to make the consequences clear?